src/Sylius/Component/Core/Calculator/ProductVariantPriceCalculatorInterface.php

```diff
@@ -25,4 +25,9 @@ interface ProductVariantPriceCalculatorInterface
      * @throws MissingChannelConfigurationException when price for given channel does not exist
      */
     public function calculate(ProductVariantInterface $productVariant, array $context): int;
+
+    /**
+     * @throws MissingChannelConfigurationException when price for given channel does not exist
+     */
+    public function calculateOriginal(ProductVariantInterface $productVariant, array $context): int;
```

Comment:
This workaround allowed me to use `calculateOriginal` method from `src/Sylius/Component/Core/OrderProcessing/OrderPricesRecalculator.php`.
In `src/Sylius/Component/Core/Calculator/ProductVariantPricesCalculatorInterface.php` we overwrite this function so it shouldn't break anything

Reply:
Could you explain what specific reason do you need to add this method to this interface? Because this change is a BC break, you extend an interface that can be used by end applications.

Reply:
Firstly, this interface is depreacted. Secondly, as @GSadee mentioned, it is BC break. Should be reverted